dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -217,32 +239,52 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					}
 
 					return (
-						<Section
-							key={collection.id}
-							title={collection.displayName}
-							showTopBorder={index > 0}
-							padContent={false}
-							centralBorder="partial"
-							url={collection.href}
-							ophanComponentLink={ophanComponentLink}
-							ophanComponentName={ophanName}
-							containerName={collection.collectionType}
-							containerPalette={collection.containerPalette}
-							toggleable={true}
-							sectionId={collection.id}
-							showDateHeader={collection.config.showDateHeader}
-							editionId={front.editionId}
-							treats={collection.treats}
-						>
-							<DecideContainer
-								trails={trails}
-								index={index}
-								groupedTrails={collection.grouped}
-								containerType={collection.collectionType}
+						<>
+							<Section
+								key={collection.id}
```

Comment:
The `key` prop should be assigned to the outermost JSX element returned by a `map` – Fixed in #7971